utils/filters.go

```diff
@@ -193,12 +193,12 @@ type FormatFilter struct {
 
 // ParseFormatParameters parses a list of formatOptions to a slice of FormatFilter.
 // If the format option has a filter, that will be the value in the struct and the filterRows boolean will be true.
-func ParseFormatParameters(formatOptions []string) []FormatFilter {
+func ParseFormatParameters(formatOptions []string, filterRows bool) []FormatFilter {
 	var parsedFilters []FormatFilter
 	for _, filter := range formatOptions {
 		filterNameAndValue := strings.SplitN(filter, "=", 2)
 		formatFilter := FormatFilter{column: filterNameAndValue[0], filterRows: false}
-		if len(filterNameAndValue) >= 2 {
+		if filterRows && len(filterNameAndValue) >= 2 {
```

Comment:
Should we return an error if `!filterRows` and `len(filterNameAndValue) >= 2`?

Reply:
Hmm I'm not sure. For example, in the status command, the format does not filter. That said, if the user inserts `--format name=root`, should we return an error because of that? We could just ignore everything after `=`, that's the current behavior in the Python client. Do you think this should be changed?

Otherwise, do we return an error because it includes an `=` or because the string `name=root` isn't a valid filter?

Reply:
It just feels odd that the user can specify `--format name=root` and we simply ignore `root`, but I guess we can leave it like this for compatibility with r-client 😄